pkg/minikube/constants/constants.go

```diff
@@ -68,11 +54,29 @@ const (
 	RemoteOpenShiftCAPath  = "/var/lib/minishift/openshift.local.config/master/ca.crt"
 )
 
-var ConfigFilePath = MakeMiniPath("config")
```

Comment:
@LalatenduMohanty yes this we can use for integration purpose I think. Not used currently.